Dockerfile.redhat-sso

```diff
@@ -13,6 +13,10 @@ COPY redhat-sso/modules/ /opt/eap/modules/system/layers/openshift/
 COPY redhat-sso/standalone-openshift.xml /opt/eap/standalone/configuration/standalone-openshift.xml
 COPY redhat-sso/datasource-common.sh /opt/eap/bin/launch/datasource-common.sh
 
+RUN curl https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.22/mysql-connector-java-8.0.22.jar \
+    -o /opt/eap/modules/system/layers/openshift/com/mysql/main/mysql-connector-java.jar
+RUN curl https://repo1.maven.org/maven2/com/oracle/database/jdbc/ojdbc8/21.4.0.0.1/ojdbc8-21.4.0.0.1.jar \
```

Comment:
Better define an ARG to centralize version numbers. Postgres is already included by default in both, right?

Reply:
I put the whole URLs as ARGs and I used docker-compose extension fields to share them between the 2 services.
Yes, PostgreSQL is already included, still 2 slightly different versions: 42.2.3 in redhat-sso and 42.2.5 in keycloak